msrestazure/azure_active_directory.py

```diff
@@ -544,7 +544,7 @@ def signed_session(self):
             raw_token = self._adal_method(*self._args, **self._kwargs)
         except adal.AdalError as err:
             # pylint: disable=no-member
-            if (hasattr(err, 'error_response') and ('error_description' in err.error_response)
+            if (getattr(err, 'error_response') and ('error_description' in err.error_response)
```

Comment:
Not very good at Python yet, but should it be `getattr(err, 'error_response', None)`? Otherwise the code will still throw when the `error_response` is missing.

Reply:
After thnking about this a little more than I did at first, I think the correct test is:

```python
if 'AADSTS70008:' in getattr(err, 'error_response', {}).get('error_description', ""):
```